### DISCUSSION TITLE
Multivariate unsupervised anomaly detection in time series data

### DISCUSSION THREAD

**Discussion by Erkan-KIYAK** [TDAmeritrade/stumpy#684] [Help: Best Practices]
My data consists of sensor data of some machines (temp, current, vibration etc. and I want to make a Predictive Maintenance. The only way to detect problems is anomaly detection for my data. The data is not labeled (unsupervised), has a timestamp info (time series), has many features (multivariate) and has no target feature (dependent variable). Does stumpy has any solution for multivariate unsupervised anomaly detection in time series data? I've searched through whole web and couldnt find any solution that fits my situation 100%. Stumpy looks like have a potential but coudnt see the exact answer. Could you please help me?

**Comment by NimaSarajpoor** [comment]
@fteller 
So, you do not know if a (continuous set of) timestamp is ~anomaly~ corresponding to a failure in system or not? is that correct?
